src/DoocsSpectrum.cc

```diff
@@ -99,7 +101,7 @@ namespace ChimeraTK {
     // send the current value to the device
     D_spectrum::read();
     modified = false;
-    if(_processArray->isWriteable()) {
+    if(this->get_access() == 1) { // property is writeable
       sendToDevice();
       // set DOOCS time stamp, workaround for DOOCS bug (get() always gives current time stamp if no timestamp is set,
       // which breaks consistency check in ZeroMQ subscriptions after the 4 minutes timeout)
```

Comment:
Why are the otherProperties not updated here, but in updateDoocsBuffer?

Reply:
Also in `sendToDevice()`...